make-release.sh

```diff
@@ -166,11 +166,12 @@ fetchAndCheckout ()
   git fetch origin "${bBRANCH}:${bBRANCH}"; git checkout "${bBRANCH}"
 }
 
-# unlike in che-plugin-registry, here we just need to update the VERSION file
-updateVersionFile () {
+updateVersion () {
```

Comment:
since this now updates multiple files, why not add a comment explaining which two files are updated? You've made this code LESS easily read than before :(